doc/reference/kernel/other/interrupts.rst

```diff
@@ -227,7 +230,7 @@ The following code defines and enables an ISR.
     #define MY_DEV_PRIO  2       /* device uses interrupt priority 2 */
     /* argument passed to my_isr(), in this case a pointer to the device */
     #define MY_ISR_ARG  DEVICE_GET(my_device)
-    #define MY_IRQ_FLAGS 0       /* IRQ flags. Unused on non-x86 */
+    #define MY_IRQ_FLAGS 0       /* IRQ flags (unused here) */
```

Comment:
"(unused here)" is hard to understand, especially because the `MY_IRQ_FLAGS` define is used further down.
I suggest to remove it.

Reply:
So you mean just keep comment `/* IRQ flags */`?

This is a bit confusing in general, as we have two types of flags here. One is this flag passed as the last parameter in `IRQ_CONNECT`. And then we have another "flag" used in the internal `struct _isr_list` mentioned further down below. Those internal flags are supposed to be prefixed with `ISR_FLAG_*` (I only found `ISR_FLAG_DIRECT` being used in the code).

Reply:
While that is confusing I don't see how "(unused here)" would clear that up.

IRQ_CONNECT/IRQ_DIRECT_CONNECT only has one valid flag I think, and that is IRQ_ZERO_LATENCY.